vault/resource_secrets_sync_gcp_destination.go

```diff
@@ -22,6 +22,7 @@ var gcpSyncWriteFields = []string{
 	consts.FieldCredentials,
 	consts.FieldSecretNameTemplate,
 	consts.FieldCustomTags,
+	consts.FieldProjectID,
```

Comment:
Is this field returned from Vault upon a read on the destination endpoint? If so, we should also add this param to the `gcpSyncReadFields` list below

Reply:
You are right. I added it on read as well